environments.json

```diff
@@ -4140,6 +4140,30 @@
     "short": "DUK 2.3",
     "platformtype": "engine",
     "release": "2017-09-23",
+    "obsolete": true
```

Comment:
This is more than 4 years old, shouldn't it be "very" instead of true?

Reply:
Thank you for reviewing! OK, I was not sure if it is very obsolete or just obsolete.